release-notes/6.0/supported-os.json

```diff
@@ -0,0 +1,411 @@
+                    ],
+                    "unsupported-versions": [
+                        "11",
+                        "10.15"
+                    ],
+                    "notes": [
+                        "The iOS and tvOS simulators are supported on macOS Arm64 and x64.",
+                        "The x64 emulator (Rosetta 2) is supported on macOS Arm64.",
+                        "Mac Catalyst apps are supported on macOS Arm64 and x64."
+                    ]
+                },
+                {
+                    "id": "tvos",
+                    "name": "tvOS",
+                    "link": "https://developer.apple.com/tvos/",
+                    "lifecycle": "https://support.apple.com/apple-tv/",
+                    "architectures": [
+                        "Arm64"
+                    ],
+                    "supported-versions": [
+                        "17",
+                        "16",
+                        "15"
+                    ],
+                    "unsupported-versions": [
+                        "12"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Linux",
+            "distributions": [
+                {
+                    "id": "alpine",
+                    "name": "Alpine",
+                    "link": "https://alpinelinux.org/",
+                    "lifecycle": "https://alpinelinux.org/releases/",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "3.20",
+                        "3.19",
+                        "3.18",
+                        "3.17"
+                    ],
+                    "unsupported-versions": [
+                        "3.16",
+                        "3.15",
+                        "3.14",
+                        "3.13",
+                        "3.12"
+                    ]
+                },
+                {
+                    "id": "debian",
+                    "name": "Debian",
+                    "link": "https://www.debian.org/",
+                    "lifecycle": "https://wiki.debian.org/DebianReleases",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "12",
+                        "11"
+                    ],
+                    "unsupported-versions": [
+                        "10"
+                    ]
+                },
+                {
+                    "id": "fedora",
+                    "name": "Fedora",
+                    "link": "https://fedoraproject.org/",
+                    "lifecycle": "https://fedoraproject.org/wiki/End_of_life",
+                    "architectures": [
+                        "Arm32",
```

Comment:
Fedora-built .NET packages are not available for arm32 on Fedora. Though Microsoft-built are: https://dotnet.microsoft.com/en-us/download/dotnet/6.0

Reply:
@omajid Is that also true for .NET 8 & 9? If yes, we should attach a note about that on all files.